recipes_source/recipes/timer_quick_start.py

```diff
@@ -0,0 +1,461 @@
+# are malloc, free, and various PyTorch utilities. However there is also a lot
+# of extraneous path junk cluttering things up. For that, you can call the
+# `as_standardized` method which will make a best effort to strip extraneous
+# parts of the symbol name:
+
+print(s0.as_standardized().stats())
+
+###############################################################################
+# .. code-block:: none
+#    :caption: Output
+#
+#     <torch.utils.benchmark.utils.valgrind_wrapper.timer_interface.FunctionCounts object at 0x7f40c0ec7c50>
+#       41137  ???:_int_free
+#       26975  ???:_int_malloc
+#       20400  ???:__tls_get_addr
+#       19800  build/../aten/src/ATen/TensorIterator.cpp:at::TensorIteratorBase::compute_types(at::TensorIteratorConfig const&)
+#       13500  ???:malloc
+#       11300  build/../c10/util/SmallVector.h:at::TensorIteratorBase::fast_set_up(at::TensorIteratorConfig const&)
+#       10000  build/../aten/src/ATen/TensorIterator.cpp:at::TensorIteratorBase::fast_set_up(at::TensorIteratorConfig const&)
+#       9200  build/../aten/src/ATen/record_function.cpp:at::shouldRunRecordFunction(bool*)
+#       9200  ???:free
+#         ...
+#         100  build/../c10/core/ScalarType.h:c10::computeDispatchKey(c10::optional<c10::ScalarType>, c10::optional<c10::Layout>, c10::optional<c10::Device>)
+#         100  build/../aten/src/ATen/core/dispatch/DispatchKeyExtractor.h:at::Tensor c10::Dispatcher::call< ... (at::Tensor const&, at::Tensor const&)> const&, at::Tensor const&, at::Tensor const&) const'2
+#         100  build/../aten/src/ATen/core/dispatch/DispatchKeyExtractor.h:at::Tensor c10::Dispatcher::call< ... r (at::Tensor const&, at::Tensor const&)> const&, at::Tensor const&, at::Tensor const&) const
+#         100  build/../aten/src/ATen/core/boxing/impl/WrapFunctionIntoFunctor.h:c10::impl::wrap_kernel_func ... const&, at::Tensor const&)>::call(c10::OperatorKernel*, at::Tensor const&, at::Tensor const&)
+#         100  /usr/include/c++/8/bits/stl_function.h:at::detail::empty_cpu(c10::ArrayRef<long>, c10::option ... ::Layout>, c10::optional<c10::Device>, c10::optional<bool>, c10::optional<c10::MemoryFormat>)
+#         100  /usr/include/c++/8/bits/atomic_base.h:void at::native::DispatchStub<void (*)(at::TensorIterator&), at::native::mul_stub>::operator()<at::TensorIterator&>(c10::DeviceType, at::TensorIterator&)
+#         100  /usr/include/c++/8/bits/atomic_base.h:c10::TensorImpl::~TensorImpl()
+#         100  /usr/include/c++/8/bits/atomic_base.h:at::native::mul(at::Tensor const&, at::Tensor const&)
+#         100  /home/taylorrobie/local/miniconda3/envs/ab_ref/lib/python3.7/site-packages/torch/include/c10/util/intrusive_ptr.h:c10::intrusive_ptr_target::release_resources()
+#
+#     Total: 587972
+#
+
+###############################################################################
+# `CallgrindStats.stats()` produces a `FunctionCounts` object, which is really
```

Comment:
this explanation belongs above, where you are first calling `stats()`?

Reply:
Done.